docs/web-apps/automated-testing/pre-run-executables.md

```diff
@@ -0,0 +1,308 @@
+
+```
+desired_capabilities['prerun'] = {
+  'executable':'https://gist.githubusercontent.com/saucyallison/3a73a4e0736e556c990d/raw/d26b0195d48b404628fc12342cb97f1fc5ff58ec/disable_fraud.sh',
+    'background': 'false'
+}
+```
+
+## Downloading Files to a VM Prior to Testing
+
+You can use a pre-run executable script to download files from a public location to the Sauce Labs virtual machine running your tests. This topic contains example scripts for downloading remote files on different operating systems, and details configuring the prerun capability in your tests.
+
+<Tabs
+  defaultValue="macOS"
+  values={[
+    {label: 'macOS', value: 'macOS'},
+    {label: 'Windows', value: 'Windows'},
+    {label: 'Windows XP', value: 'Windows XP'},
+    {label: 'Linux', value: 'Linux'},
+  ]}>
+
+<TabItem value="macOS">
+
+### OS X 10.6, 10.8, 10.9, 10.10
+This shell script will fetch the file at the URL and save it to **/tmp/file.txt**.
+
+```js
+#!/bin/bash
+curl -o /tmp/file.txt http://mywebsite.com/file.txt
+```
+</TabItem>
+
+<TabItem value="Windows">
+
+### Windows 7, 8, 8.1
+This batch file accomplishes the same thing as the OS X curl method, but using **bitsadmin.exe** since Windows doesn't ship with curl.
+
+```js
+@echo off
+bitsadmin.exe /transfer "JobName" http://mywebsite.com/file.txt C:\Users\Administrator\Desktop\file.txt
+```
+</TabItem>
+
+<TabItem value="Windows XP">
+
+### Windows XP
+This batch file creates a VBScript file, **dl.vbs**, which will perform the download, and then runs it:
+
+```js
+@echo off
+
+echo strFileURL="http://mywebsite.com/file.txt" > C:\dl.vbs
+echo strHDLocation = "C:\file.csv" >> C:\dl.vbs
+echo Set objXMLHTTP = CreateObject("MSXML2.XMLHTTP") >> C:\dl.vbs
+echo objXMLHTTP.open "GET", strFileURL, false >> C:\dl.vbs
+echo objXMLHTTP.send() >> C:\dl.vbs
+echo If objXMLHTTP.Status = 200 Then >> C:\dl.vbs
+echo Set objADOStream = CreateObject("ADODB.Stream") >> C:\dl.vbs
+echo objADOStream.Open >> C:\dl.vbs
+echo objADOStream.Type = 1 'adTypeBinary >> C:\dl.vbs
+echo objADOStream.Write objXMLHTTP.ResponseBody >> C:\dl.vbs
+echo objADOStream.Position = 0 'Set the stream position to the start >> C:\dl.vbs
+echo Set objFSO = Createobject("Scripting.FileSystemObject") >> C:\dl.vbs
+echo If objFSO.Fileexists(strHDLocation) Then objFSO.DeleteFile strHDLocation >> C:\dl.vbs
+echo Set objFSO = Nothing >> C:\dl.vbs
+echo objADOStream.SaveToFile strHDLocation >> C:\dl.vbs
+echo objADOStream.Close >> C:\dl.vbs
+echo Set objADOStream = Nothing >> C:\dl.vbs
+echo End if >> C:\dl.vbs
+echo Set objXMLHTTP = Nothing >> C:\dl.vbs
+
+cscript.exe C:\dl.vbs
+```
+
+</TabItem>
+
+<TabItem value="Linux">
+
+### Linux
+This shell script downloads **file.txt** at **mywebsite.com** to the **/tmp** directory.
+
+```js
+#!/bin/bash
+wget -O /tmp/file.txt http://mywebsite.com/file.txt
+```
+</TabItem>
+</Tabs>
+
+## Set Prerun Capability
```

Comment:
Ah...I think I get this now. Were the H2 sections in this doc originally different pages in the wiki, so each one includes a section on how to set the `prerun` capability for that flavor of the executable? That makes sense, but I don't think we want to do that here, all in the same page. How do you feel about pulling it out and making it its own H2?